Libraries/LibCore/EventLoop.cpp

```diff
@@ -233,20 +233,10 @@ EventLoop::EventLoop()
         ASSERT(rc == 0);
         s_event_loop_stack->append(this);
 
-        auto rpc_path = String::format("/tmp/rpc.%d", getpid());
-        rc = unlink(rpc_path.characters());
-        if (rc < 0 && errno != ENOENT) {
-            perror("unlink");
-            ASSERT_NOT_REACHED();
+        if (!s_rpc_server) {
+            if (!start_rpc_server())
+                dbg() << "Failed to start a RPC server";
```

Comment:
Would be nice to know who failed to start _an_ RPC server.

Also, the commit message needs some love

Reply:
`dbg()` outputs the name and PID of the process, so you get it automatically.

Reply:
> Also, the commit message needs some love

what do you mean, it looks fine to me?

Reply:
I was thinking `"EventLoop: Failed to do whatever"`, but that was just a nitpick to get the "the commit message needs some love" nitpick attached to the commit :)

Reply:
"LibCore: Do not assert we can start the RPC server" reads like a few things to me, I guess you can make it better by adding a "that" after assert

just pointing that out since the sentence gave me a pause.

Reply:
> I was thinking "EventLoop: Failed to do whatever"

Hmm, I was assuming everyone knows what is the RPC server in context of Serenity, but sure, let's spell it out.

> "the commit message needs some love" nitpick attached to the commit

Some more details, please? :slightly_frowning_face:

Reply:
Inded, thanks

Reply:
> I was assuming everyone knows what is the RPC server in context of Serenity

Think of the lives of the poor tech support people you'll save doing this :P 

> Some more details, please?

You just omitted a "that" in the message and my brain refused to parse it for a few seconds.
"LibCore: Do not assert that we can start the RPC server"

that's all.